src/pages/Home/Home.js

```diff
@@ -26,7 +26,7 @@ class Home extends Component {
   }
 
   static defaultProps = {
-    classes: [{ name: 'Class A' }, { name: 'Class B' }],
+    classes: [{ name: 'CS 4500: Software Development' }, { name: 'ANTH 5513: Ancient Greek History' }],
```

Comment:
This is going to change a lot in the redux layer, so nothing will be matching anymore from home page -> enter time page